static/css/legacy_community.css

```diff
@@ -588,14 +585,6 @@ h2:after {
 }
 
 @media (max-width:750px) {
-
-
-  .twittercol1 {
-    width: 95%;
-    float: left;
-    padding: 2%
-  }
-
```

Comment:
Removed because of the responsive styling should be handled by X (Twitter) widget itself rather than actual numbers here.